virtcontainers/network.go

```diff
@@ -244,6 +247,39 @@ func (endpoint *VirtualEndpoint) Detach(netNsCreated bool, netNsPath string) err
 	})
 }
 
+// HotAttach for the virtual endpoint uses hot plug device
+func (endpoint *VirtualEndpoint) HotAttach(h hypervisor, netNsPath string) error {
+	networkLogger().Info("Hot attaching virtual endpoint")
+	if err := xsetupVMNetwork(endpoint); err != nil {
```

Comment:
It'd be better if we could use `xconnectVMNetwork(&(endpoint.NetPair), true)` here instead. Unless there's a real blocker, could you tell more about it ?